skiko/src/jvmMain/kotlin/org/jetbrains/skiko/SkikoProperties.kt

```diff
@@ -3,41 +3,41 @@ package org.jetbrains.skiko
 // TODO maybe we can get rid of global properties, and pass SkiaLayerProperties to Window -> ComposeWindow -> SkiaLayer
 @Suppress("SameParameterValue")
 internal object SkikoProperties {
-    val vsyncEnabled: Boolean by property("skiko.vsync.enabled", default = true)
+    val vsyncEnabled: Boolean = property("skiko.vsync.enabled", default = true)
 
     /**
      * If vsync is enabled, but platform can't support it (Software renderer, Linux with uninstalled drivers),
      * we enable frame limit by the display refresh rate.
      */
-    val vsyncFramelimitFallbackEnabled: Boolean by property(
+    val vsyncFramelimitFallbackEnabled: Boolean = property(
         "skiko.vsync.framelimit.fallback.enabled", default = true
     )
 
-    val fpsEnabled: Boolean by property("skiko.fps.enabled", default = false)
-    val fpsPeriodSeconds: Double by property("skiko.fps.periodSeconds", default = 2.0)
+    val fpsEnabled: Boolean = property("skiko.fps.enabled", default = false)
+    val fpsPeriodSeconds: Double = property("skiko.fps.periodSeconds", default = 2.0)
 
     /**
      * Show long frames which is longer than [fpsLongFramesMillis].
      * If [fpsLongFramesMillis] isn't defined will show frames longer than 1.5 * (1000 / displayRefreshRate)
      */
-    val fpsLongFramesShow: Boolean by property("skiko.fps.longFrames.show", default = false)
+    val fpsLongFramesShow: Boolean = property("skiko.fps.longFrames.show", default = false)
 
-    val fpsLongFramesMillis: Double? by property("skiko.fps.longFrames.millis", default = null)
+    val fpsLongFramesMillis: Double? = property("skiko.fps.longFrames.millis", default = null)
 
-    val renderApi: GraphicsApi by lazy {
+    val renderApi: GraphicsApi get() {
         val environment = System.getenv("SKIKO_RENDER_API")
         val property = System.getProperty("skiko.renderApi")
-        if (environment != null) {
+        return if (environment != null) {
             parseRenderApi(environment)
         } else {
             parseRenderApi(property)
         }
     }
 
-    private fun parseRenderApi(text: String?): GraphicsApi {
+    internal fun parseRenderApi(text: String?): GraphicsApi {
         when(text) {
-            "SOFTWARE" -> return GraphicsApi.SOFTWARE
-            "DIRECT_SOFTWARE" -> return GraphicsApi.DIRECT_SOFTWARE
+            "AWT_SOFTWARE" -> return GraphicsApi.AWT_SOFTWARE
```

Comment:
Didn't we agree that it will be FAST_SOFTWARE and COMPAT_SOFTWARE?

Reply:
Thanks for noticing! Forget to revert in this PR. Done